app/main/util/render_utils.py

```diff
@@ -57,36 +58,37 @@ def get_download_filename(file):
 
 
 def manage_static_file(file, file_extension):
-    """Manage the file in the static directory."""
+    """Manage the file in a temporary directory."""
```

Comment:
It took me a while to figure out the purpose of the manage_static function.

It seems like the function is writing the file stream from s3 object to a temp file on the lambda.

Im assuming so the file can be served by the html template / universal viewer which assues the file exists there?

Could we use the returned path and pass it to the render_teplate which we can pass down to record.html so that we dont have to hardcode

                {% if file_extension == "pdf" %}
                    <object data="{{ url_for('static', filename='files/temp_file.pdf') }}"
                            width="100%"
                            height="700px">
                        <p>Please enable JS to view this record.</p>
                    </object>
                {% elif file_extension == "png" or file_extension == "jpg" %}
                    <object data="{{ url_for('static', filename='files/temp_file.png') }}"

that way we dont couple this route to the html

Also currently Id assume the template needs to change anyway as it is still hardcoded to static, not temp?